src/objects/Hierarchy_Settings__c.object

```diff
@@ -77,6 +77,28 @@
         <type>Checkbox</type>
     </fields>
     <fields>
+        <fullName>Default_Enrolled_Student_RecordType_Id__c</fullName>
```

Comment:
Can we follow the existing naming format used in Household_Addresses_RecType__c, and name these something like Student_RecType__c and Faculty_RecType__c?